tools/utility/dockerise/platform.py

```diff
@@ -25,7 +25,7 @@ def selected(repository):
 
         names = [
             tag.split(":")[-1]
-            for tag in img_info[0]["RepoTags"]
```

Comment:
Double list comprehension makes my brain hurt. Could you please split the assignment of `names` into smaller blocks? I am having trouble _comprehending_ what this does at all. (I wouldn't class this as scope creep, because the replacement you have offered takes the complexity over a threshold where my brain hurts.)

Reply:
This is more of a filter than a double list comprehension.
But I agree with kip in this case, easier to add a new line that splits out your `/`